templates/Bookmarks.html

```diff
@@ -1,9 +1,83 @@
 {% extends 'base.html' %}
 
 {% block head %}
+<style>
+  main {
+    background-image: url(/static/images/signup-background2.png);
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+  }
 
+  body {
+    overflow-y: auto;
+  }
+
+  .bm-table {
+    width: 100%;
+    max-width: 100%;
+    margin-left: 1rem;
+    margin-right: 1rem;
+  }
+
+  .bm-table td, .bm-table th {
+    text-align: center;
+    padding: 8px;
+  }
+
+  .bm-table td, .bm-table th, .bm-table tcol {
+    border-radius: 5px;
+    -moz-border-radius: 5px;
+    -webkit-border-radius: 5px;
+  }
+
+  .bm-table thead th {
+    color: #ffffff;
+    background: #69779C;
+  }
+
+  .bm-table thead th:nth-child(odd) {
+    background: #3A3E5E;
+    color: #ffffff;
+  }
+
+  .bm-table tr:nth-child(even) {
+    background: #69779C;
+    color: #ffffff;
+  }
+
+  .bm-table tr:nth-child(odd) {
+    background: #F8F8F8;
+  }
+</style>
```

Comment:
I will likely move these styles into a 'tableStyles.css' - but for now this is solid!